docs/guides/javascript.md

```diff
@@ -107,21 +107,22 @@ openapi: 3.0.0
 `
 
 const spectral = new Spectral();
-spectral.registerFormat('oas2', isOpenApiv2);
-spectral.registerFormat('oas3', isOpenApiv3);spectral.loadRuleset('spectral:oas3') // spectral:oas2 for OAS 2.0 aka Swagger
+spectral.registerFormat('oas2', isOpenApiv2); // spectral:oas2 for OAS 2.0 aka Swagger
```

Comment:
```suggestion
spectral.registerFormat('oas2', isOpenApiv2); // spectral:oas2 for OpenAPI v2.0
```

We can do better than saying "oas2 = OAS 2" and we mention swagger plenty elsewhere.

Reply:
Btw I know you didnt add this im just noticing it now 😅

Reply:
@philsturgeon Fixed